src/main/java/flaxbeard/steamcraft/tile/TileEntitySteamCharger.java

```diff
@@ -85,15 +84,14 @@ public void updateEntity() {
         if (this.worldObj.isRemote) {
             if (this.getStackInSlot(0) != null) {
                 if (this.isCharging) {
-                    this.worldObj.spawnParticle("smoke", xCoord + 0.5F, yCoord + 0.5F,
-                      zCoord + 0.5F, (Math.random() - 0.5F) / 12.0F, 0.0F,
-                      (Math.random() - 0.5F) / 12.0F);
+                    this.worldObj.spawnParticle("smoke", xCoord + 0.5F, yCoord + 0.5F, zCoord + 0.5F,
+                            (Math.random() - 0.5F) / 12.0F, 0.0F, (Math.random() - 0.5F) / 12.0F);
                 }
             }
         } else {
             if (this.getStackInSlot(0) != null) {
-                if (this.getStackInSlot(0).getItem() == SteamcraftItems.steamcellEmpty &&
-                  this.getSteamShare() > Config.steamCellCapacity) {
+                if (this.getStackInSlot(0).getItem() == SteamcraftItems.steamcellEmpty
+                        && this.getSteamShare() > Config.steamCellCapacity) {
```

Comment:
the `&&` should actually be on the other line, like `== SteamcraftItems.steamcellEmpty &&`.